### PR TITLE
update cloud functions package.json

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**
Move to node 12
Remove unused dependancies

### CODE REVIEW

File: apps/backend-functions/src/package.json [hunk 1]
```diff
@@ -42,11 +27,7 @@
     "faker":"5.1.0",
     "generate-password": "^1.4.2",
     "jwplatform": "0.0.5",
-    "line-reader": "^0.4.0",
```
Comment (phpgeekfr) [inline apps/backend-functions/src/package.json:45]: Can also be removed from main package.json i guess

File: apps/backend-functions/src/package.json [hunk 2]
```diff
@@ -42,11 +27,7 @@
...
     "lodash": "^4.17.11",
-    "request": "^2.88.0",
```
Comment (phpgeekfr) [inline apps/backend-functions/src/package.json:47]: same

File: apps/backend-functions/src/package.json [hunk 3]
```diff
@@ -4,24 +4,10 @@
   "license": "MIT",
   "main": "./main.js",
   "engines": {
-    "node": "10"
+    "node": "12"
```
Comment (phpgeekfr) [inline apps/backend-functions/src/package.json:7]: @george43g can this be a problem inside CI ? DockerFile is using node 10

**Review by phpgeekfr — APPROVED** [review]

File: apps/backend-functions/src/package.json (hunk 3 above)
Comment (george43g) [reply apps/backend-functions/src/package.json:7]: not really... it's only if we're running jest tests of cloud functions, then it's useful to use the same version of node for the tests (CI container) as is used in the actual cloud functions

File: package.json [hunk 4]
```diff
@@ -3,7 +3,7 @@
   "version": "0.0.2",
   "license": "MIT",
   "engines": {
-    "node": "10"
+    "node": "12"
```
Comment (george43g) [inline package.json:6]: why is this here at all?

**Review by george43g — APPROVED** [review]
nice work!
